README.md

```diff
@@ -72,4 +60,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] your licence link is broken , kindly fix it by changing `[MIT](./LICENSE)` to `[MIT](./MIT.MD)`